frontend/app/model/MembersOnlyContent.scala

```diff
@@ -0,0 +1,31 @@
+package model
+
+import com.gu.contentapi.client.model.{Asset, Content}
+
+case class MembersOnlyContent(content: Content) {
```

Comment:
Do you still want all this? I thought we said we'd put this in the Controller as move it if it's used in more than one place?